osf/migrations/0055_update_metaschema_active.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-09-14 14:32
+from __future__ import unicode_literals
+import logging
+
+from django.db import migrations
+
+from osf.models import MetaSchema
+from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+
+
+logger = logging.getLogger(__file__)
+
+ACTIVE_METASCHEMAS = [
+    'Prereg Challenge',
+    'Open-Ended Registration',
+    'AsPredicted Preregistration',
+    'OSF-Standard Pre-Data Collection Registration',
+    'Replication Recipe (Brandt et al., 2013): Pre-Registration',
+    'Replication Recipe (Brandt et al., 2013): Post-Completion',
+    "Pre-Registration in Social Psychology (van 't Veer & Giner-Sorolla, 2016): Pre-Registration",
+]
+
+def update_metaschema_active(*args, **kwargs):
+    for schema in MetaSchema.objects.all():
+        if schema.name not in ACTIVE_METASCHEMAS or schema.schema_version != LATEST_SCHEMA_VERSION and schema.active:
+            logger.info('{} was active, setting to inactive.'.format(schema))
+            schema.active = False
```

Comment:
This is OK but can be improved by using `.update`:

```python
MetaSchema.objects.exclude(name__in=ACTIVE_METASCHEMAS).exclude(schema_version=LATEST_SCHEMA_VERSION).update(active=False)
```

In the near future, all models will have a `modified` field that automatically gets set on `.save()`. Using `.update` prevents that from getting updated.